hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerStateManagerImpl.java

```diff
@@ -86,7 +87,7 @@
 public final class ContainerStateManagerImpl
```

Comment:
@adoroszlai , do we need similar striped lock in org.apache.hadoop.hdds.scm.container.ContainerManagerImpl class to be changed as well ?

Reply:
Thanks for pointing out `ContainerManagerImpl`.

`ContainerManagerImpl` uses a single lock, not a set of "managed" locks.  It may be possible to improve that by adding a striped lock for some operations that apply to a single container, but that's a different task.